homeassistant/components/ambient_station/sensor.py

```diff
@@ -61,13 +63,13 @@ def unit_of_measurement(self):
 
     async def async_update(self):
         """Fetch new state data for the sensor."""
-        new_state = self._ambient.stations[
-            self._mac_address][ATTR_LAST_DATA].get(self._sensor_type)
-
-        if self._sensor_type == TYPE_SOLARRADIATION:
-            # Ambient's units for solar radiation (illuminance) are
-            # W/m^2; since those aren't commonly used in the HASS
-            # world, transform them to lx:
-            self._state = round(float(new_state)/0.0079)
+        if self._sensor_type == TYPE_SOLARRADIATION_LX:
+            # If the user requests the solarradiation_lx sensor, use the
```

Comment:
Should we remove monitored_conditions option in light of https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md?

Reply:
Yes! 👍